### PR TITLE
Stripe payment

### PR DESCRIPTION
**Pull request by joshuabello2550**
Summary:

- Added stripe ui payment
- Connected Parse server with stripe API keys
- Need to test if payments actually go through and are displayed on Stripe account

Test plan:
- Ui is displayed

### CODE REVIEW

File: app/src/main/java/com/example/xpvehicles/activities/AddVehicleActivity.java [hunk 1]
```diff
@@ -146,7 +146,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (requestCode == CAPTURE_IMAGE_ACTIVITY_REQUEST_CODE) {
             if (resultCode == RESULT_OK) {
                 // by this point we have the camera photo on disk
-                Bitmap takenImage = BitmapFactory.decodeFile(photoFile.getAbsolutePath());
+//                Bitmap takenImage = BitmapFactory.decodeFile(photoFile.getAbsolutePath());
```
Comment (leixiangwu) [inline app/src/main/java/com/example/xpvehicles/activities/AddVehicleActivity.java:149]: delete?

File: app/src/main/java/com/example/xpvehicles/activities/VehicleDetailsActivity.java [hunk 2]
```diff
@@ -88,7 +98,7 @@ private void setVehicleSwipeListener(ViewPager2 viewPager, int totalNumberOfImag
             @Override
             public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
                 super.onPageScrolled(position, positionOffset, positionOffsetPixels);
-                TextView tvVehicleImagePosition =  findViewById(R.id.tvVehicleImagePosition);
+                TextView tvVehicleImagePosition = findViewById(R.id.tvVehicleImagePosition);
```
Comment (leixiangwu) [inline app/src/main/java/com/example/xpvehicles/activities/VehicleDetailsActivity.java:101]: should we find this view in onCreate?

File: app/src/main/java/com/example/xpvehicles/activities/VehicleDetailsActivity.java [hunk 3]
```diff
@@ -201,21 +213,52 @@ private Boolean checkValidDates() {
...
             public void onClick(View v) {
                 Boolean validDates = checkValidDates();
                 if (validDates) {
-                    _User currentUser = (_User) ParseUser.getCurrentUser();
-                    RentVehicle rentVehicle = createRentVehicle(currentUser);
-                    requestVehicle(currentUser, rentVehicle);
-                    finish();
+                    HashMap<String, Object> params = new HashMap<>();
+                    params.put("amount", (Integer) orderTotal * 100);
+                    params.put("currency", "usd");
```
Comment (leixiangwu) [inline app/src/main/java/com/example/xpvehicles/activities/VehicleDetailsActivity.java:224]: create constants?

File: app/src/main/java/com/example/xpvehicles/activities/AddVehicleActivity.java (hunk 1 above)
Comment (joshuabello2550) [reply app/src/main/java/com/example/xpvehicles/activities/AddVehicleActivity.java:149]: Currently unsure if I will need it to fix the issue where images aren't displayed in AddVehicleActivity

**Pull request merged**